src/sick_tim_common_tcp.cpp

```diff
@@ -42,17 +42,19 @@
 namespace sick_tim
 {
 
-SickTimCommonTcp::SickTimCommonTcp(const std::string &hostname, const std::string &port, AbstractParser* parser)
+SickTimCommonTcp::SickTimCommonTcp(const std::string &hostname, const std::string &port, const std::string &timelimit, AbstractParser* parser)
 :
     SickTimCommon(parser),
     socket_(io_service_),
     deadline_(io_service_),
     hostname_(hostname),
-    port_(port)
+    port_(port),
+    timelimit_(timelimit)
 {
     // Set up the deadline actor to implement timeouts.
     // Based on blocking TCP example on:
     // http://www.boost.org/doc/libs/1_46_0/doc/html/boost_asio/example/timeouts/blocking_tcp_client.cpp
+
```

Comment:
Please don't add unrelated new lines. (Rational: this pull request is quite small, so it's not that important, but if you submit larger requests, it makes them harder to review.)